roles/manifest/README.md

```diff
@@ -8,9 +8,9 @@ Role Variables
 
 This role supports the [Common Role Variables](https://github.com/theforeman/foreman-ansible-modules/blob/develop/README.md#common-role-variables).
 
-- `manifest_path`: Path to subscription Manifest file on Ansible target host. When using `manifest_download`, it is first downloaded to this location from the Red Hat Customer Portal before being uploaded to the Foreman server.
-- `manifest_download`: Whether to first download the Manifest from the Red Hat Customer Portal. Defaults to `False`.
-- `manifest_uuid`: UUID of the Manifest to download, corresponding to a [Subscription Allocation](https://access.redhat.com/management/subscription_allocations) defined on your Red Hat account. Required when `manifest_download` is `True`.
+- `foreman_manifest_path`: Path to subscription Manifest file on Ansible target host. When using `manifest_download`, it is first downloaded to this location from the Red Hat Customer Portal before being uploaded to the Foreman server.
+- `foreman_manifest_download`: Whether to first download the Manifest from the Red Hat Customer Portal. Defaults to `False`.
+- `foreman_manifest_uuid`: UUID of the Manifest to download, corresponding to a [Subscription Allocation](https://access.redhat.com/management/subscription_allocations) defined on your Red Hat account. Required when `manifest_download` is `True`.
 - `rhsm_username`: Your username for the Red Hat Customer Portal. Required when `manifest_download` is `True`.
 - `rhsm_password`: Your password for the Red Hat Customer Portal. Required when `manifest_download` is `True`.
```

Comment:
I didn't rename those two, but I guess we should.

`foreman_rhsm_username` just sounds so wrong…

Reply:
We definitely should. `whatever_username` and `whatever_password` are the primary examples for collisions when we don't namespace properly.

Reply:
```suggestion
- `rhsm_username`: Your username for the Red Hat Customer Portal. Required when `manifest_download` is `True`.
- `rhsm_password`: Your password for the Red Hat Customer Portal. Required when `manifest_download` is `foreman_true`.
```

Just kidding. ;) But should "true" not be lowercase in yaml?

Reply:
should, yes, `ansible-lint` will yell at you, but it accepts both, can update when I touch this, sure